package.json

```diff
@@ -9,8 +9,8 @@
     "build": "vite build",
     "serve": "vite preview",
     "lint": "eslint . --ext .js,.ts,.vue",
-    "test:unit": "vitest run --silent",
-    "test:coverage": "vitest run --coverage"
+    "test:unit": "vitest run",
+    "test:coverage": "vitest run --silent"
```

Comment:
You need to leave the coverage option in there.

Reply:
That's the option for reporting how well the unit tests cover the different code paths.

Eg,
```
----------------------------------------------------------|----------|----------|----------|----------|-------------------|
File                                                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
----------------------------------------------------------|----------|----------|----------|----------|-------------------|
All files                                                 |    46.98 |    11.14 |     14.8 |     47.6 |                   |
 src                                                      |      100 |      100 |      100 |      100 |                   |
  App.vue                                                 |      100 |      100 |      100 |      100 |                   |
 src/components                                           |      100 |      100 |      100 |      100 |                   |
  tabs.vue                                                |      100 |      100 |      100 |      100 |                   |
```